src/components/BlockingViews/BlockingView.tsx

```diff
@@ -77,22 +111,40 @@ function BlockingView({
         );
     }
 
+    function renderSubtitle() {
```

Comment:
this is just a bit different from many other files in the repo. Let's change both `renderContent` and `renderSubtitle` to just use `useMemo` and return the rendered content instead?